vignettes/metadata_analysis.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Metadata analysis using fulltext"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Metadata analysis using fulltext}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
```

Comment:
we're not using rmarkdown here, copy the format of the other vignettes - see https://raw.githubusercontent.com/ropensci/fulltext/master/vignettes/getting_fulltext.Rmd